src/modules/claims/claims.service.ts

```diff
@@ -278,12 +278,21 @@ export class ClaimsService {
         });
     }
 
-    async rejectClaimRequest({ id, requesterDID }: { id: string; requesterDID: string }) {
+    async rejectClaimRequest({
+        id,
+        requesterDID,
+        rejectionReason,
+    }: {
+        id: string;
+        requesterDID: string;
+        rejectionReason?: string;
+    }) {
         const message: IClaimRejection = {
             id,
             requester: requesterDID,
             claimIssuer: [this._signerService.did],
             isRejected: true,
+            rejectionReason,
```

Comment:
@JGiter This approach makes sense to me, so the cache-server just needs to accept an additional `rejectionReason` string in the rejection message